models/database.py

```diff
@@ -31,13 +32,15 @@ class User(Base, UserMixin):
     lastname = Column(String(40), nullable=False)
     email = Column(String(120), nullable=False)
     image_file = Column(String(240), nullable=False, default="default.jpg")
-    password = Column(String(60), nullable=False)
+    password = Column(Text, nullable=False)
+    #location = Column(Geography(geometry_type='POINT', srid=4326))
+    #domicile_geolocation = Column(Geography(geometry_type='POINT', srid=4326))
     messages = relationship("Messages", backref="author", lazy=True)
     profile = relationship("Profile", uselist=False, back_populates="user")
     user_criteria = relationship("UserCriteria", uselist=False, back_populates="user")
 
     def __repr__(self):
-        return f"User('{self.name}', '{self.lastname}','{self.email}', '{self.password}', '{self.image_file}')"
+        return f"User('{self.name}', '{self.lastname}','{self.email}', '{self.password}', '{self.image_file}', '{self.location}')"
 
 
 class Messages(Base, UserMixin):

```

Comment:
UserMixin here is quite suspicious -_- and in every module different than User xD

Reply:
Also mixins should be placed before base clases